### ISSUE TITLE
Mursi 473 is mislinked to Murle

### ISSUE THREAD

**Issue by Taivo55** [cldf-datasets/phoible#8]
Mursi (UPSID 473) is linked to Glottolog Murle (murl1244).  It should be linked to Glottolog Mursi (murs1242).  In addition, Mursi (UPSID 473) lists two ultimate bibliographic sources for Mursi:  Turton and Bender 1976 (which is the only source listed in UPSID for this language) and Arensen 1982.  Arensen is actually a source for Murle, but isn't listed at murle (AA 793).

Note:  Please don't think I'm trying to pick apart your excellent phonological reference site.  I'm using it as a database for some of my own research (and will, of course, cite it appropriately) and am simply noticing things that pop out at me.

**Comment by Taivo55** [comment]
As an update, I looked at the on-line UPSID and notice that they list Mursi with Murle as an alternate name.  I was using Maddieson's earlier version of UPSID in "Patterns of Sounds".  The online version also lists both Arensen 1982 and Turton and Bender 1976 as sources for Mursi.  Glottolog, however, correctly separates Mursi and Murle into two languages (that not are even sisters in the Surmic family, but cousins).  Arensen 1982 is included in the bibliography for Murle there and Turton and Bender 1976 is included in the bibliography for Mursi.  So clearly this error predates Phoible and can be attributed to the last version of UPSID

**Comment by bambooforest** [comment]
@Taivo55 -- no offense taken! We appreciate very much any and all notices for corrections and I'd be happy to add you to the list of credits (https://phoible.org/credits) for your contributions. 

As you've correctly pointed out, the last decades of linguistic fieldwork have led to more precise language identification, as illustrated by the the splitting of ISO 639-3 codes.

This case though does seem to be a choice between assigning Murle (murl1244) vs Mursi (UPSID 473), where Maddieson used a bibliographic source from each language in his typologization of the phonological inventory.

In general, we try to leave UPSID (and any of its mistakes) as is for preservation purposes.

Maddieson has been working on an expanded and corrected version called Lapsyd and there: 

http://www.lapsyd.ddl.cnrs.fr/lapsyd/index.php?data=view&code=240

the entry for Murle includes a third reference, which Glottolog also lists under Murle:

https://glottolog.org/resource/reference/id/18341

So, two of three references are from Murle.

I'm inclined to leave it as is and perhaps put a note somewhere on phoible (dev), e.g. the README page for the UPSID raw-data.

What do you think @drammock?

Regarding AA 793, this is extracted from the Alphabets of Africa and cited as such:

https://archive.org/details/rosettaproject_mur_ortho-1/mode/1up
http://sumale.vjf.cnrs.fr/phono/PhonologieN.php

**Comment by Taivo55** [comment]
If I might comment about the confusion, the original UPSID, found in Maddieson's "Patterns of Sound" (pg 307), was actually correct in citing just one source for "Mursi", Turton and Bender 1976, which was unequivocally about Mursi, without listing "Murle" as an alternative name.  I checked and they don't even mention "Murle".  The original UPSID inventory is of Murse.  The last version of UPSID has the same inventory, but, without comment, includes the Arensen 1982 source (which I have not seen).  The link you gave me to lapsyd was to the Murle entry, which is not problematic in either Phoible or Lapsyd.  All the references are accurate and, as far as I can tell, the inventories match the Murle sources.  So we can leave the Murle inventory aside.  The only problem is in the Mursi entry here in Phoible (UPSID 473).

If you look up Mursi in Lapsyd you arrive at [http://www.lapsyd.ddl.cnrs.fr/lapsyd/index.php?data=view&code=724] which includes the original UPSID reference to Turton and Bender 1976 (Mursi only) as well as the new reference to Yigezu 2001, which is also cited at the Lapsyd entry for Murle.  This Mursi inventory in Lapsyd matches not only the original UPSID inventory (labelled "Mursi"), but the Phoible inventory drawn from it.  In other words, Lapsyd recognizes the original UPSID inventory as Mursi, including the Turton and Bender 1976 reference (which is not referenced at the Murle entry), but also the validity of separating Murle from Mursi.  The problem arose with the last version of UPSID where, inexplicably, the Arensen reference to Murle was added to Mursi.

But even the issue is just one of preserving the historical record, the last version of UPSID still labels this inventory (Mursi UPSID 473) as Mursi and includes "Murle" only as an alternative name.  It still preserves the original Turton and Bender 1976 reference and uses their inventory as the inventory.  Using the "alternative name" as the link in Phoible when Mursi has been recognized as a separate language throughout the history of the Surmic family is probably an error that needs to be corrected.

So to summarize, Murle (AA 793), is fine and requires no fixes.  It was never used in UPSID, either the original version or the later versions.

Mursi (UPSID 473) should link to Glottolog murs1242 (ISO 639-3 muz), Mursi, since the inventory and its history are clearly tied to Mursi and not to Murle, except in the singular reference to "Murle" as an alternative name in the last version of UPSID

**Comment by Taivo55** [comment]
Thank you also for offering to add me to your list of credits.  My IRL name is John E. McLaughlin (I'm an associate professor at Utah State University with a PhD in Linguistics from the University of Kansas)

**Comment by drammock** [comment]
> even if the issue is just one of preserving the historical record, the last version of UPSID still labels this inventory (Mursi UPSID 473) as Mursi and includes "Murle" only as an alternative name. It still preserves the original Turton and Bender 1976 reference and uses their inventory as the inventory.

Makes sense

> Mursi (UPSID 473) should link to Glottolog murs1242 (ISO 639-3 muz), Mursi, since the inventory and its history are clearly tied to Mursi and not to Murle, except in the singular reference to "Murle" as an alternative name in the last version of UPSID

I'm convinced that we should make this change. If I understand correctly, UPSID isn't the locus of this mistake, it's our linkage from UPSID to glottolog.

**Comment by Taivo55** [comment]
I have to apologize for making this sound like something of monumental proportions.  The only thing that actually needs changing is the link to Mursi in Glottolog rather than to Murle on this single entry (Mursi UPSID 473)

**Comment by drammock** [comment]
no apology necessary @Taivo55. We very much appreciate it when researchers with interest or expertise in a particular family help us find and fix our mistakes :)

**Comment by bambooforest** [comment]
@Taivo55 - thanks! fixed in:

https://github.com/phoible/dev/pull/262
